scripts/generate-abis.ts

```diff
@@ -25,14 +25,14 @@ const assetsDir = path.join(
 );
 
 // Path to directory where ABIs will be written
-const outputDir = path.join(process.cwd(), 'src', 'abis', 'safe');
+const outputDir = path.join(process.cwd(), 'dist', 'abis', 'safe');
```

Comment:
You'd need to bundle this in the docker image too 😬

Reply:
I _think_ I did so in 0e7c2b494c9f3084fcf805103fc9f43a8920658a. Please let me know if it's correct.